styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
+	background-color:#093C60;
+	color:white;
+    padding: 5% 13%;
```

Comment:
This padding is really agressive and seems like it's causing layout issues a bit. Maybe dial it down a tad?